src/main/java/picard/analysis/MergeableMetricBase.java

```diff
@@ -39,24 +39,32 @@
 import java.util.List;
 
 /**
- * An extension of MetricBase that knows how to merge-by-adding fields that are appropriately annotated. It also provides an interface
- * for calculating derived fields (and an annotation that informs that said fields are derived). Finally, it also allows for an annotation
- * that suggests that a field will be used as an ID and thus merging will simply require that these fields are equal.
- *
- * merge-by-adding is only enabled for the following types: int, Integer, float, Float, double, Double, short, Short, long, Long, byte, Byte.
+ * An extension of MetricBase that knows how to merge-by-adding fields that are appropriately annotated ({@link MergeByAdding}). It also provides an interface
+ * for calculating derived fields {@link #calculateDerivedFields()}  (and an annotation that informs that said fields are derived {@link NoMergingIsDerived}). Finally, it also allows for an annotation
+ * that suggests that a field will be used as an ID and thus merging will simply ensure that these fields are equal {@link MergeByAssertEquals}. Other annotations are available,
+ * though they all currently imply that no <it>implicit</it> action will be taken {@link MergingIsManual} and {@link NoMergingKeepsValue}.
+ *<p>
+ * {@link MergeByAdding} is only enabled for the following types: int, Integer, float, Float, double, Double, short, Short, long, Long, byte, Byte.
```

Comment:
If you use this form instead of `{@link MergeByAdding MergeByAdding}` then it will show up as `MergableMetricBase.MergeByAdding` in the javadoc. I think the shorter form is easier to read.

To see what your javadoc will look like, put the cursor on the class name and hit control-J (or use View/Quick Documentation).